plugins/labhub.py

```diff
@@ -1,7 +1,258 @@
+                             filter(lambda x: (x.full_name.split('/')[0] ==
+                                               self.GH_ORG_NAME),
+                                    self.IGH.write_repositories)}
+            self.gl_repos = {repo.full_name.split('/')[1]: repo for repo in
+                             filter(lambda x: (x.full_name.split('/')[0] ==
+                                               self.GL_ORG_NAME),
+                                    self.IGL.write_repositories)}
+            self.REPOS = {**self.gh_repos, **self.gl_repos}
+        except RuntimeError:
+            self.log.error('Either of GH_TOKEN or GL_TOKEN is not set')
+
+    @property
+    def TEAMS(self):
+        return self._teams
+
+    @TEAMS.setter
+    def TEAMS(self, new):
+        self._teams = new
+
+    @re_botcmd(pattern=r'(?:(?:invite)|(?:inv))\s+(\w+)\s*(?:to)\s+(\w+)')
+    def invite_cmd(self, msg, match):
+        """
+        Invite given user to given team. By default it invites to
+        "newcomers" team.
+        """
+        invitee = match.group(1)
+        inviter = msg.frm.nick
+
+        team = (self.GH_ORG_NAME + ' newcomers' if match.group(2) is None
+                else match.group(2))
+
+        self.log.info('{} invited {} to {}'.format(inviter, invitee, team))
+
+        if self.TEAMS[self.GH_ORG_NAME + ' maintainers'].is_member(invitee):
+            valid_teams = ['newcomers', 'developers', 'maintainers']
+            if team.lower() not in valid_teams:
+                return 'Please select from one of the ' + ', '.join(valid_teams)
+            team_mapping = {
+                'newcomers': self.GH_ORG_NAME + ' newcomers',
+                'developers': self.GH_ORG_NAME + ' developers',
+                'maintainers': self.GH_ORG_NAME + ' maintainers'
+            }
+
+            # send the invite
+            self.TEAMS[team_mapping[team.lower()]].invite(invitee)
+            return self.INVITE_SUCCESS[team.lower()].format(invitee)
+        else:
+            return ('@{}, you are not a maintainer, only maintainers can invite'
+                    ' other people. Nice try :poop:'.format(inviter))
+
+    def callback_message(self, msg):
+        """Invite the user whose message includes the holy 'hello world'"""
+        if re.search(r'hello\s*,?\s*world', msg.body, flags=re.IGNORECASE):
+            user = msg.frm.nick
+            if not self.TEAMS[self.GH_ORG_NAME + ' newcomers'].is_member(user):
+                # send the invite
+                self.send(msg.frm,
+                          self.INVITE_SUCCESS['newcomers'].format(user))
+                self.TEAMS[self.GH_ORG_NAME + ' newcomers'].invite(user)
+
+    @re_botcmd(pattern=r'(?:new|file) issue ([\w-]+?)(?: |\n)(.+?)(?:$|\n((?:.|\n)*))',  # Ignore LineLengthBear, PyCodeStyleBear
+               flags=re.IGNORECASE)
+    def create_issut_cmd(self, msg, match):
+        '''Create issues on GitHub and GitLab repositories.'''
```

Comment:
yeah, this one was because QuotesBear is buggy for oneliner double quotes docstrings, and would have to add ignore comment everywhere. Thanks for pushing this, I'll add the ignores now :sweat_smile: